lib/modules/accountSwitcher.js

```diff
@@ -28,12 +23,12 @@ module.options = {
 	},
 	accounts: {
 		type: 'table',
-		addRowText: '+add account',
+		addRowText: i18n('accountSwitcherAddAccount'),
```

Comment:
I'll fix up the renderer in this PR and merge it, you don't need to do anything ATM